src/io/CavityWriter.h

```diff
@@ -5,17 +5,34 @@
 
 #include "plugins/PluginBase.h"
 
+class CavityEnsemble;
```

Comment:
I like that you use forward declaration, but it's confusing to me that
std::map<unsigned, CavityEnsemble> _mcav;
compiles. As far as I understood from
https://stackoverflow.com/questions/38898935/forward-declaration-of-objects-with-stl-containers
it is not guaranteed by the standard and, hence, we might run into trouble eventually, if using different compilers or on weird clusters. Spricht was dagegen auf Pointers umzusteigen:
std::map<unsigned, * CavityEnsemble> _mcav;
?

Reply:
Changed as requested 8c04960